source/Calamari.Tests/Fixtures/PackageRetention/JsonJournalRepositoryFixture.cs

```diff
@@ -53,12 +54,29 @@ public void WhenCalamariPackageRetentionJournalPathDoesNotExist_ThenTheJournalIs
             variables.Get(KnownVariables.Calamari.PackageRetentionJournalPath).Returns((string) null);
             variables.Get(TentacleVariables.Agent.TentacleHome).Returns(homeDir);
 
-            var repository = new JsonJournalRepository(TestCalamariPhysicalFileSystem.GetPhysicalFileSystem(), Substitute.For<ISemaphoreFactory>(), variables);
+            var repository = new JsonJournalRepository(TestCalamariPhysicalFileSystem.GetPhysicalFileSystem(), Substitute.For<ISemaphoreFactory>(), variables, Substitute.For<ILog>());
 
             var expectedPath = Path.Combine(homeDir, JsonJournalRepository.DefaultJournalName);
 
             repository.Commit();
             Assert.IsTrue(File.Exists(expectedPath));
         }
+
+        [Test]
+        public void WhenThereIsAnErrorReadingTheJournal_ThenTheJournalIsRenamed()
+        {
+            const string invalidJson = @"[{""a"",}]";
+            var journalPath = Path.Combine(TentacleHome, "PackageRetentionJournal.json");
+
+            File.WriteAllText(journalPath, invalidJson);
+
+            var variables = Substitute.For<IVariables>();
+            variables.Get(KnownVariables.Calamari.PackageRetentionJournalPath).Returns(journalPath);
+
+            // Load()
```

Comment:
Do we need this comment?

Reply:
Don't think so. Initially had it as a "this is where the load happens" but the test is pretty small and I think the name of it explains what it's doing well enough.